Misc/NEWS.d/next/Security/2019-05-21-23-20-18.bpo-35907.NC_zNK.rst

```diff
@@ -0,0 +1,2 @@
+CVE-2019-9948: Avoid file reading as disallowing the unnecessary URL scheme in
```

Comment:
Would you like to incorporate comment by @gpshead in https://github.com/python/cpython/pull/13474/files#r286726212 ?

Reply:
I will first fix the NEWS entry in master: https://github.com/python/cpython/pull/13523

And then update other branches and this PR.

Reply:
Fixed